tests/desktop/test_users_account.py

```diff
@@ -75,7 +75,7 @@ def test_user_can_access_the_view_profile_page(self, mozwebqa):
 
     @pytest.mark.native
     @pytest.mark.login
-    def test_hide_email_checkbox_works(self, mozwebqa):
+    def test_hide_email_checkbox(self, mozwebqa):
```

Comment:
There is a merge conflict with this file, and you'll find that the issue is that this line needs to change to:

`def test_hide_email_checkbox(self, mozwebqa, existing_user):`